### PR TITLE
ref(platform content): Hide pages and sections with missing snippets

### PR DESCRIPTION
**Pull request by untitaker**
To fix https://github.com/getsentry/sentry-docs/issues/3107, we are hiding
platform sections and sometimes entire pages that are missing snippets. At a
later point we can go back and un-hide those pages once the corresponding SDK
feature is correctly documented. Right now these pages are generally useless
because they basically say that feature is supported, but don't tell you how to
use it.

### CODE REVIEW

File: src/platforms/common/configuration/releases.mdx [hunk 1]
```diff
@@ -88,8 +88,12 @@ In order to monitor release health, the SDK sends session data.
 
 A session represents the interaction between the user and the application. Sessions contain a timestamp, a duration, a status (if the session was OK or if it crashed), and are always linked to a release. Most Sentry SDKs can manage sessions automatically.
 
+<PlatformSection notSupported={["rust"]}>
```
Comment (Swatinem) [inline src/platforms/common/configuration/releases.mdx:91]: we do have this in rust: https://docs.rs/sentry/latest/sentry/struct.ClientOptions.html#structfield.auto_session_tracking

File: src/platforms/common/enriching-events/breadcrumbs.mdx [hunk 2]
```diff
@@ -53,7 +53,7 @@ You can choose from the following breadcrumb log levels: `"fatal"`, `"critical"`
 
 <PlatformContent includePath="enriching-events/breadcrumbs/automatic-breadcrumbs" />
 
-<PlatformSection notSupported={["unreal"]}>
+<PlatformSection notSupported={["unreal", "rust", "elixir"]}>
```
Comment (Swatinem) [inline src/platforms/common/enriching-events/breadcrumbs.mdx:56]: also supported in rust: https://docs.rs/sentry/latest/sentry/struct.ClientOptions.html#structfield.before_breadcrumb

File: src/platforms/common/usage/sdk-fingerprinting.mdx [hunk 3]
```diff
@@ -9,6 +9,8 @@ notSupported:
   - native.wasm
   - native.minidump
   - unreal
+  - rust
```
Comment (Swatinem) [inline src/platforms/common/usage/sdk-fingerprinting.mdx:12]: events have a fingerprint in rust, and you can mess with that in `before_send`: https://docs.rs/sentry/latest/sentry/types/protocol/v7/struct.Event.html#structfield.fingerprint

**Comment by untitaker** [comment]
as discussed with @Swatinem offline we're going to merge this so that the stub pages disappear, and his team will eventually add examples + reenable those pages

**Comment by untitaker** [comment]
I'm going to merge this PR today since nobody has raised strong objections.

**Review by bruno-garcia — APPROVED** [review]
:rocket:

**Pull request merged**